code/game/area/space_station_13_areas.dm

```diff
@@ -1036,6 +1045,16 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	airlock_wires = /datum/wires/airlock/service
 	sound_environment = SOUND_AREA_STANDARD_STATION
 
+/area/office/clown
```

Comment:
Are these necessary?

Reply:
It's just a goofy thing for Chimera, I think it's funny